boot/env.ml

```diff
@@ -38,11 +38,28 @@ let theories_dir = "theories"
 let plugins_dir = "plugins"
 let prelude = Filename.concat theories_dir "Init/Prelude.vo"
 
+let coqbin =
+  Util.canonical_path_name (Filename.dirname Sys.executable_name)
```

Comment:
Can we avoid exporting `canonical_path_name` at all? It is a function with weird semantics, as IIRC was introduced to workaround makefile bugs (so IMHO better to get rid of this fix the real problem)

Reply:
beyond the scope of this PR

Reply:
What I mean is move it along `coqbin` to `boot/env.ml` so you don't have to export it in the .mli file?

As the function wasn't exported before, it seems to me to be in scope, isn't it?

Reply:
Actually exporting it opens competition with the current export in `CUnix`, IMHO we don't want that.